configman/config_manager.py

```diff
@@ -45,21 +45,50 @@
 import functools
 import warnings
 
-import configman as cm
-import converters as conv
-import config_exceptions as exc
-import value_sources
-import def_sources
-
 #==============================================================================
 # for convenience define some external symbols here - some client modules may
 # import these symbols from here rather than their origin definition location.
 # PyFlakes may erroneously flag some of these as unused
-from option import Option, Aggregation
-from dotdict import DotDict, DotDictWithAcquisition, iteritems_breadth_first
-from namespace import Namespace
-from required_config import RequiredConfig  # used elsewhere - do not remove
-from config_file_future_proxy import ConfigFileFutureProxy
+from configman import (
+    command_line
+)
+from configman.converters import (
+    to_string_converters,
+)
```

Comment:
This an area where the inconistency really bothers me.  They ought all be one way or all be the other.  Putting them on one line very quickly runs afoul of another infamous stylistic tenet, so I find myself defaulting to the multi-line method.

In your example you actually make my point.  The "time to reconsider" should never happen, it is a waste of both time and consideration.  To quote PEP20: "There should be one-- and preferably only one --obvious way to do it"